cd/manager/jobmanager/jobManager.go

```
@@ -38,8 +39,18 @@ func NewJobManager(cache manager.Cache, db manager.Database, d manager.Deploymen
 			maxAnchorJobs = parsedMaxAnchorWorkers
 		}
 	}
+	minAnchorJobs := manager.DefaultCasMinAnchorWorkers
+	if configMinAnchorWorkers, found := os.LookupEnv("CAS_MIN_ANCHOR_WORKERS"); found {
+		if parsedMinAnchorWorkers, err := strconv.Atoi(configMinAnchorWorkers); err == nil {
+			minAnchorJobs = parsedMinAnchorWorkers
+		}
+	}
+	// Sanity check the counts
+	if minAnchorJobs > maxAnchorJobs {
+		minAnchorJobs = maxAnchorJobs
```

I feel like this should throw an error. Should not allow you to proceed if you misconfigure this.